src/components/carousel/index.jsx

```diff
@@ -283,18 +283,16 @@ const Carousel = ({
 
 		if (document.fullscreenEnabled) {
 			if (!document.fullscreenElement) {
-				fullScreenElement.requestFullscreen().then(() => setIsFullScreen(true));
+				fullScreenElement.requestFullscreen();
 			} else {
-				document.exitFullscreen().then(() => setIsFullScreen(false));
+				document.exitFullscreen();
 			}
 		} else {
 			// safari needs prefix
 			// eslint-disable-next-line no-lonely-if
 			if (document.webkitFullscreenEnabled) {
 				if (!document.webkitFullscreenElement) {
-					fullScreenElement.webkitRequestFullscreen().then(() => setIsFullScreen(true));
-				} else {
```

Comment:
The code `} else { document.webkitExitFullscreen(); }` should still be here, otherwise people on iOS won't be able to exit the fullscreen view.

Reply:
@vgalatro Fixed this